src/validation/tripRequestValidation.js

```diff
@@ -147,6 +147,32 @@ export default class TripRequestValidation {
     return true;
   }
 
+  /**
+  * Validates Trip stats Request paramenters
+  *
+  * @param {object} body - The request object
+  * @param {object} res - The request response object
+  * @returns {object} - returns an object (error or response).
+  */
+  static statsRequest(body) {
+    const schema = {
+      startDate: Joi.date()
+        .format('YYYY-MM-DD')
+        .required()
+        .error(TripRequestValidation.validateTripStats('startDate')),
+      endDate: Joi.date()
+        .format('YYYY-MM-DD')
+        .min(body.startDate)
+        .required()
+        .error(TripRequestValidation.validateTripStats('endDate'))
+    };
```

Comment:
Can you please shed some light on my your validations are written this way?

Reply:
Apart from validating a specific date format, this validation ensures that the **endDate** is not less than **startDate** before querying the database